ksql-engine/src/main/java/io/confluent/ksql/planner/plan/FlatMapNode.java

```diff
@@ -66,7 +64,11 @@ public FlatMapNode(
     this.analysis = Objects.requireNonNull(analysis);
     this.functionRegistry = functionRegistry;
     this.finalSelectExpressions = buildFinalSelectExpressions();
-    outputSchema = buildLogicalSchema(source.getSchema());
+    outputSchema = StreamFlatMapBuilder.buildSchema(
```

Comment:
This is just bridge code. Planning to drop this in a follow-up. We shouldn't need to get the schema from the logical plan nodes.